assets/konnected.css

```diff
@@ -856,6 +856,16 @@ iframe#bwpFrame {
 }
 
 
+.custom-badge {
+  color: rgb(var(--color-base-text));
+  font-weight: 600;
+  --badge-corner-radius: 0;
+}
+
+.has-cbadge .custom-badge {
+  display: inline-block !important;
+}
+
```

Comment:
don't need custom style
badge styles should be consistent (see thread: https://konnected-io.slack.com/archives/C046E5GMTCK/p1696347140664139)